src/bot/commands/random/dog.ts

```diff
@@ -0,0 +1,212 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Message } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+import { Url } from 'url';
+const entities = require("html-entities").AllHtmlEntities;
+
+let breeds: string[] = [
```

Comment:
There's enough breeds here to consider putting them in their own file and just importing it.